### PR TITLE
Storages: add metrics of read thread

### PR DESCRIPTION
**Pull request by JinheLin**
### What problem does this PR solve?

Issue Number: ref #6834

### What is changed and how it works?
image

```commit-message

```

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp [hunk 1]
```diff
@@ -169,6 +169,7 @@ void SegmentReadTaskPool::finishSegment(const SegmentReadTaskPtr & seg)
     if (pool_finished)
     {
         q.finish();
+        LOG_INFO(log, "pool_id={} finished", pool_id);
```
Comment (Lloyd-Pottiger) [inline dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp:172]: Is it necessary?

**Review by CalvinNeo — APPROVED** [review]
lgtm

File: dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp (hunk 1 above)
Comment (Lloyd-Pottiger) [reply dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp:172]: How about recording the overall time of `readOneBlock`, and printing it in destructor?

Comment (Lloyd-Pottiger) [reply dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp:172]: `total_dmfile_read_time_ns` is the sum of all segments. The duration between the first time call `readOneBlock` and the last time `readOneBlock` returned is the real `TablesSan` duration.

And we may want to know the time cost in delta.

**Review by Lloyd-Pottiger — APPROVED** [review]

File: dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp (hunk 1 above)
Comment (Lloyd-Pottiger) [reply dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp:172]: ok

**Pull request merged**